Framework/AnalysisSupport/src/AODJAlienReaderHelpers.cxx

```diff
@@ -299,13 +299,19 @@ AlgorithmSpec AODJAlienReaderHelpers::rootFileReaderCallback()
           std::string nextFileRead = info.file->GetPath();
           if (currentFileRead != nextFileRead) {
             currentFileRead = nextFileRead;
+            std::string monitoringInfo(currentFileRead);
+            monitoringInfo += ",";
+            monitoringInfo += std::to_string(info.file->GetSize());
 #if __has_include(<TJAlienFile.h>)
             auto alienFile = dynamic_cast<TJAlienFile*>(info.file);
             if (alienFile) {
-              /// FIXME: get the JAlien stats
+              monitoringInfo += ",";
+              monitoringInfo += alienFile->GetSE();
```

Comment:
I wonder if the problem with the metrics is not that `GetSE()` returns something which the monitoring backend does not understand as a properly formatted string? I will check.

Reply:
At least the line which appears with LOGP looks proper: E.g. "alien:///alice/data/2015/LHC15o/000244918/pass5_lowIR/PWGZZ/Run3_Conversion/96_20201013-1346_child_1/0028/AO2D.root:/,631838549,ALICE::CERN::EOS"

Reply:
If you don't mind, merge this, we can parse the log output if we don't get the logging up